e2e/client.go

```diff
@@ -0,0 +1,26 @@
+package e2e
+
+import (
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+)
+
+// NewClientConfigForTest returns a config configured to connect to the api server
+func NewClientConfigForTest() (*rest.Config, error) {
+	loader := NewClientConfigLoadingRulesForTest()
+	clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loader, &clientcmd.ConfigOverrides{})
+	return clientConfig.ClientConfig()
+}
+
+// NewClientConfigLoadingRulesForTest returns a ClientConfigLoadingRules object that should find the kubeconfig
+// for tests in some of the expected places.
+func NewClientConfigLoadingRulesForTest() *clientcmd.ClientConfigLoadingRules {
+	rules := clientcmd.NewDefaultClientConfigLoadingRules()
+	// KUBERNETES_CONFIG might only be a OperatorSDK thing
```

Comment:
not sure we need 2 env vars (KUBECONFIG is what normal e2e and cli use) for specifying the same thing but I don't care that much if we do

Reply:
I'm good with removing `KUBERNETES_CONFIG` as I didn't actually see it in use except for some (possibly) operator SDK related code.

Reply:
I don't think we use operator sdk, hopefully

Reply:
we don't